tabpy-server/server_tests/test_config.py

```diff
@@ -87,7 +87,7 @@ def test_config_file_present(self, mock_os, mock_path_exists, mock_psws, mock_sh
         mock_os.getenv.assert_has_calls(getenv_calls, any_order=True)
         self.assertEqual(settings['log_level'], 'WARNING')
         self.assertEqual(settings['port'], 1234)
-        self.assertEquals(settings['server_version'], tabpy_server.__version__)
+        self.assertEqual(settings['server_version'], open('VERSION').read().strip())
```

Comment:
Any reason to move away from the dunder? PEP-8 actually encourages the former use and that's what I've come to expect with most pacakges

https://www.python.org/dev/peps/pep-0008/#id24

Reply:
Does this actually pass? I would think the `open('VERSION')` expression should fail